utils/crud/mongodb.go

```diff
@@ -9,12 +9,12 @@ import (
 )
 
 // MongoClaimsCollection is the name of the claims collection.
-const MongoClaimsCollection = "cnab_claims"
+const MongoCollectionPrefix = "cnab_"
```

Comment:
tiny note: we can remove the `claims` specific section from the comment above

Reply:
I am terrible at updating the comments. 🤦‍♀ Thanks, I'll get that fixed.